src/Data/Fin/Base.agda

```diff
@@ -77,11 +77,24 @@ fromℕ<″ zero    (ℕ.less-than-or-equal refl) = zero
 fromℕ<″ (suc m) (ℕ.less-than-or-equal refl) =
   suc (fromℕ<″ m (ℕ.less-than-or-equal refl))
 
--- raise m "i" = "m + i".
+-- canonical liftings of smaller i:Fin m to larger index Fin (n + m)
```

Comment:
This comment is correct with respect to `_↑ʳ_` but confusing with respect to `_↑ˡ_` with is jut below (because of the switch of order between `n + m` and `m + n`).

Reply:
Thanks Jacques. 
IIUC, you want *two* comments, each of which disambiguates the argument order, as it were so that 'one of them lands in Fin(n + m), while the other lands in Fin(m + n)'...
But no *actual* changes to the typechecked terms (or their names).
If that understanding is correct, I'll try to push a commit to achieve that. 
FWIW this is my first pull request evs. and consequently, I am hyper-conscious of 9the possibility/likelihood of) screwing up any 'approved' git-, never mind agda-stdlib-dev, worklfow...

Reply:
I don't necessarily care about the number of comments. It makes sense to have a 'global' comment and some local ones - I just want them to be correct wrt the code!  And indeed, no code changes seem to be needed.

Reply:
I see.

Reply:
(and sorry for polluting the Issue discussion with these conversation points. Ugh. So many new mental models to learn all in one fell swoop...

Reply:
There's absolutely no problem with having discussions in the PR comments :+1: I think the current record for the stdlib is something like 200+ comments on a single particularly divisive PR.